fix from upstream for warnings:
gcc-4.3.2/libjava/classpath/native/fdlibm/dtoa.c:913: warning: incompatible implicit declaration of built-in function 'free'
gcc-4.3.2/libjava/classpath/native/fdlibm/dtoa.c:918: warning: incompatible implicit declaration of built-in function 'free'

2008-06-18  Ralf Wildenhues  <Ralf.Wildenhues@gmx.de>

	* native/fdlibm/dtoa.c: include stdlib.h, for 'free'.

Index: libjava/classpath/native/fdlibm/dtoa.c
===================================================================
--- libjava/classpath/native/fdlibm/dtoa.c	(revision 136907)
+++ libjava/classpath/native/fdlibm/dtoa.c	(revision 136908)
@@ -28,6 +28,7 @@
 
 #include "mprec.h"
 #include <string.h>
+#include <stdlib.h>
 
 static int
 _DEFUN (quorem,
